src/skeleplex/graph/skeleton_graph.py

```diff
@@ -50,6 +50,52 @@ def skeleton_graph_decoder(json_object):
     return json_object
 
 
+def make_graph_directed(graph: nx.Graph, origin=int) -> nx.DiGraph:
+    """Return a directed graph from an undirected graph.
+
+    The directed graph has the same nodes and edges as the undirected graph.
+
+    Parameters
+    ----------
+    graph : nx.Graph
+        The undirected graph to convert to a directed graph.
+    origin : int
+        The node to use as the origin node for the directed graph.
+        The origin node will have no incoming edges.
+    """
+    if isinstance(graph, nx.DiGraph):
+        print("The input graph is already a directed graph.")
```

Comment:
Can you please use the logging module? That way users can filter this message out (e.g., when they are running a pipeline). The "info" level is probably fine.

https://docs.python.org/3/library/logging.html